gitpod-io/gitpod
# [rebuild] decouple IDE env vars from registry-facade

## Description


by moving in supervisor-ide-config.json

## Related Issue(s)

Fixes #

## How to test


- Start workspace with different IDEs like VS Code Browser, Desktop and IntelliJ. Check that gp open, gp preview and code works for task and regular IDE terminals.
- Check that with gp rebuild for VS Code Browser and Desktop (opened from a file menu) gp open, gp preview and code works too.

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
